sim/druid/druid.go

```diff
@@ -242,30 +241,28 @@ func New(char core.Character, form DruidForm, selfBuffs SelfBuffs, talents proto
 
 func init() {
 	core.BaseStats[core.BaseStatsKey{Race: proto.Race_RaceTauren, Class: proto.Class_ClassDruid}] = stats.Stats{
-		stats.Health:      7237,
-		stats.Strength:    85,
-		stats.Agility:     86,
-		stats.Stamina:     98,
-		stats.Intellect:   143,
-		stats.Spirit:      159,
-		stats.Mana:        3496,
-		stats.SpellCrit:   1.85 * core.CritRatingPerCritChance, // Class-specific constant
-		stats.AttackPower: -20,                                 // accounts for the fact that the first 20 points in Str only provide 1 AP rather than 2
-		stats.MeleeCrit:   0.96 * core.CritRatingPerCritChance, // 3.56% chance to crit shown on naked character screen
-		stats.Dodge:       -1.87 * core.DodgeRatingPerDodgeChance,
+		stats.Health:    6892, // 8227 health shown on naked character (would include tauren bonus)
+		stats.Strength:  94,
+		stats.Agility:   78,
+		stats.Stamina:   99,
+		stats.Intellect: 139,
+		stats.Spirit:    161,
+		stats.Mana:      3496,                                // 5301 mana shown on naked character
+		stats.SpellCrit: 1.85 * core.CritRatingPerCritChance, // Class-specific constant
+		stats.MeleeCrit: 7.48 * core.CritRatingPerCritChance, // 8.41% chance to crit shown on naked character screen
```

Comment:
Naked character would also include the crit from base agility. Seenig how you have a slightly lower value i would guess you already account for that but asking just in case

Reply:
yea these are all verified, might be a .01% swing or rounding error but as close as i can for now